digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -96,35 +151,27 @@ export default {
             this.isLoaded = true;
           });
       }
+      this.$socket.emit('joinRoom', this.$store.getters.getSessionId);
```

Comment:
Putting this outside the `if` will have the player join the room regardless of whether the session was successfully initialized. This may also be related to a bug I'm seeing where after clicking `Create` the page just reloads and the url becomes `http://localhost:3000/?`

Reply:
yeah, I changed it to only join when form is correct but I still the `?` in the url. I think this may be related to cookies

Reply:
You should put this inside `finally`, though you should probably change it into `then` since everything in `finally` runs regardless of whether there was an error or not, and we don't want to continue if there was an error.